keyring/keyring.go

```diff
@@ -4,6 +4,7 @@ import "errors"
 
 const (
 	KeychainBackend string = "keychain"
+	SecretsBackend  string = "secrets"
```

Comment:
Was thinking about it but its not actually using libsecret since it goes through dbus via godbus. Couldn't think of a better name than libsecret for the separate go package since there are already so many named *secret